client/dashboard/src/pages/mcp/MCPDetails.tsx

```diff
@@ -519,33 +520,19 @@ export function MCPDetails({ toolset }: { toolset: Toolset }) {
         <PublicToggle isPublic={mcpIsPublic ?? false} />
       </PageSection>
       <PageSection
```

Comment:
One though. I do wonder if it's worth showing this only if the server is public. Because the logo and documentation link in a hosted page is only going to applicable for a public server.

At the same time. Maybe we always show it and just leave a note about that in description